### PR TITLE
Ajout de test de la config i18n

### PR DESCRIPTION
**Pull request by Julien-laville**
Ajout de test de la config i18n
Es6 homogénéisation 
Mise à jours de la sortie des tests

### CODE REVIEW

File: task/test.js [hunk 1]
```diff
@@ -42,9 +45,47 @@ if(!style.metadata['openmaptiles:version']) {
...
+
+/* Test i18n config file */
+
+const languageFallbacks = yaml.readSync(path.join(args.style_dir,'i18n.yml')).languageFallbacks
+
+const layersFields = [];
+let error = false;
+
+style.layers.map((layer) => {
+  if(layer['layout'] && layer['layout']['text-field'] && layer['layout']['text-field'] === '{name}') {
```
Comment (amatissart) [inline task/test.js]: Is that not a bit dangerous to hardcode the value `{name}` here ?  
What if another field is used by default in the future, such as `{name_int}` ?

Comment (Julien-laville) [reply task/test.js]: j'ata une suggestion

Comment (nlehuby) [reply task/test.js]: `layer['layout']['text-field'].startsWith('{name');` ?

ok: `{name}`, `{name:fr}`, `{name:en}`, `{name:int}`, `{name:latin} \n {name}`
ko: `{ref}`

File: task/test.js [hunk 2]
```diff
@@ -42,9 +45,47 @@ if(!style.metadata['openmaptiles:version']) {
...
   console.log(
-    'WARNING: Style is not formatted. You should use "gl-style-format"\n' +
+    'WARNING: Style is not correctly formatted. You should use "gl-style-format"\n' +
     'before commit. See https://github.com/mapbox/mapbox-gl-js/blob/master/src/style-spec/README.md');
 }
+
+
+/* Test i18n config file */
+
+const languageFallbacks = yaml.readSync(path.join(args.style_dir,'i18n.yml')).languageFallbacks
```
Comment (nlehuby) [inline task/test.js]: I needed to add `path.resolve` (to get an absolute path) to make this work.

File: task/test.js [hunk 3]
```diff
@@ -42,9 +45,47 @@ if(!style.metadata['openmaptiles:version']) {
...
+    if(layersField.status) {
+      console.log(`✓ ${layersField.id} | ${layersField.lang}`)
+    } else {
+      console.error(`❌ ${layersField.id}`)
+    }
+  });
+
+  process.exit(1);
+} else {
+  console.log(`✓ i18n config file`)
```
Comment (nlehuby) [inline task/test.js]: `✓ i18n config file is ok` would be more explicit.

File: task/test.js [hunk 4]
```diff
@@ -42,9 +45,47 @@ if(!style.metadata['openmaptiles:version']) {
...
+  }
+});
+
+if(error) {
+  console.error('ERROR I18n file config is incomplete')
+  layersFields.forEach((layersField) => {
+    if(layersField.status) {
+      console.log(`✓ ${layersField.id} | ${layersField.lang}`)
+    } else {
+      console.error(`❌ ${layersField.id}`)
```
Comment (nlehuby) [inline task/test.js]: :+1: :+1:

**Review by nlehuby — CHANGES_REQUESTED** [review]

File: task/test.js (hunk 1 above)
Comment (amatissart) [reply task/test.js]: :+1: I suppose that's a good compromise.

**Review by jbgriesner — APPROVED** [review]

File: task/build_cli.js [hunk 5]
```diff
@@ -21,6 +21,15 @@ const args = yargs
       nargs: 1
     }
   })
+  .options({
```
Comment (nlehuby) [inline task/build_cli.js:24]: new parameter => update the readme (done)

**Review by nlehuby — APPROVED** [review]

**Pull request merged**